lucille-core/src/test/resources/ElasticsearchIndexerTest/routing.conf

```diff
@@ -0,0 +1,21 @@
+indexer {
+  type: "Elasticsearch"
+  routingField: "routing"
+
+  # todo whats the point of having this list?
```

Comment:
this tells the indexer to not send the id and routing fields to elasticsearch as part of the document
these fields aren't needed as document fields in ES because ES captures them elsewhere
the test should work fine without this setting
however, it is arguably a good idea to leave this setting in place in the test to make sure that routing info is being set even when "routing" is in the list of ignoreFields

where possible we should avoid leaving TODOs/questions in the code so instead please instead leave a comment explaining that this is for